src/components/TableContent/TableContent.js

```diff
@@ -3,12 +3,14 @@ import React from "react";
 import { Link } from "../Link/Link";
 
 export function TableContent(props) {
+  const lang = props.lang?.trim?.() === "fr" ? "fr" : "en";
   const { id, sectionList } = props;
-
   return (
     <>
       <section id={id}>
-        <p className="ds-heading3">On this Page:</p>
+        <h3 className="ds-heading3">
+          {lang === "en" ? "On this Page" : "Sur cette page"}
```

Comment:
```suggestion
          {lang === "en" ? "On this page" : "Sur cette page"}
```
Sentence case here and this should be contained within an h2 else the pages won't follow proper heading hierarchy.

Reply:
Sorry I accidentally pushed this code to this branch. I've removed it and have this in a different branch: https://github.com/DTS-STN/Service-Canada-Design-System/pull/704